src/test/resources/db/authzdb.sql

```diff
@@ -0,0 +1,118 @@
+-- Roles
+--
+
+CREATE ROLE authz_reader;
+ALTER ROLE authz_reader WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD '${pgsql.test.password}';
+CREATE ROLE authz_writer;
+ALTER ROLE authz_writer WITH NOSUPERUSER INHERIT NOCREATEROLE NOCREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD '${pgsql.test.password}';
+
+--
+-- PostgreSQL database initialization
+--
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+SET default_tablespace = '';
+SET default_table_access_method = heap;
+
+--
+-- Name: items; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.items (
+    uid text NOT NULL,
+    access_level smallint DEFAULT 0 NOT NULL
+);
+
+CREATE TABLE public.origins (
+    url text NOT NULL,
+    degraded_allowed boolean DEFAULT FALSE NOT NULL
+);
+
+ALTER TABLE public.items OWNER TO postgres;
+
+ALTER TABLE public.origins OWNER TO postgres;
+
+--
+-- Name: COLUMN items.uid; Type: COMMENT; Schema: public; Owner: postgres
+--
+
+COMMENT ON COLUMN public.items.uid IS 'The unique identifier of the requested object';
+
+--
+-- Name: COLUMN items.access_level; Type: COMMENT; Schema: public; Owner: postgres
+--
+
+COMMENT ON COLUMN public.items.access_level IS 'The access level of the requested item: 0 is open, 1 is restricted to UCLA affiliates';
+
+--
+-- Name: COLUMN origins.url; Type: COMMENT; Schema: public; Owner: postgres
+--
+
+COMMENT ON COLUMN public.origins.url IS 'The URL origin of an access request';
+
+--
+-- Name: COLUMN origins.degraded_allowed; Type: COMMENT; Schema: public; Owner: postgres
+--
+
+COMMENT ON COLUMN public.origins.degraded_allowed IS 'Whether this origin allows degraded access';
+
+
+--
+-- Data for Name: items; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.items (uid, access_level) FROM stdin;
+\.
+
+--
+-- Data for Name: origins; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.origins (url, degraded_allowed) FROM stdin;
+\.
+
+--
+-- Name: items items_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.items
+    ADD CONSTRAINT items_pkey PRIMARY KEY (uid);
+
+--
+-- Name: items origins_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.origins
+    ADD CONSTRAINT origins_pkey PRIMARY KEY (url);
+
+--
+-- Name: TABLE items; Type: ACL; Schema: public; Owner: postgres
+--
+
+GRANT SELECT ON TABLE public.items TO authz_reader;
+GRANT ALL ON TABLE public.items TO authz_writer;
+
+--
+-- Name: TABLE origins; Type: ACL; Schema: public; Owner: postgres
+--
+
+GRANT SELECT ON TABLE public.origins TO authz_reader;
+GRANT ALL ON TABLE public.origins TO authz_writer;
```

Comment:
Should we have tests for the items and origins tables at this point?

Reply:
I just have the one database test at this point to confirm it was loaded. It's making the assumption if that one change from the SQL file was loaded that the rest are too.

I think we should leave more detailed tests to the tickets that are actually using these table structures. As we saw in our Josh meeting a few minutes ago, these may change as we get into the implementation (i.e., this is just my best first guess) so I think it makes more sense to add tests along with the implementations.